profiles/migrations/0005_alter_profile_options_alter_savedcompany_options_and_more.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.3 on 2023-09-06 19:48
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0004_viewedcompany'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='profile',
+            options={'ordering': ['profile_id']},
+        ),
+        migrations.AlterModelOptions(
+            name='savedcompany',
+            options={'ordering': ['company_id']},
```

Comment:
I really, really discourage you of using `Meta.ordering` on models.

While it works nicely in cases when project is small and there is not a lot of records in DB, it brings just HUGE amount of issues on bigger projects.

Ordering must be applied explicitly, _when_ needed and _where_ needed.

Reply:
To give you more context. This ordering will be applied on each DB request to the model, doesn't matter if there is a sense to do ordering (which, by the way, is not 'for free', it's pretty heavy operation on DB level).

I'm not 100% sure, but it's possible that with `Meta.ordering` on model you'll have issues trying to implement ordering in DRF filters etc.

So, just don't use this.